bucket/cairo-shell.json

```
@@ -0,0 +1,30 @@
+{
+    "version": "0.4.80",
+    "description": "Cairo Desktop Environment",
+    "homepage": "https://cairoshell.com/",
+    "license": "Apache 2.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/cairoshell/cairoshell/releases/download/v0.4.80/CairoSetup_64bit.exe#/dl.7z",
+            "hash": "b7a37dc28ec7a78f0906d92843619e431513719051e0186153779b3cf31ef2fd"
+        }
+    },
+    "bin": "CairoDesktop.exe",
+    "shortcuts": [
+        [
+            "CairoDesktop.exe",
+            "Cairo Desktop"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/cairoshell/cairoshell",
+        "regex": "tag/v([\\d.]+)"
```

Ideally find out winsparkle URL, which is used in the application

```suggestion
        "url": "https://github.com/cairoshell/cairoshell/releases",
        "regex": "download/v([\\d.]+)/"
```